docs/resources/cluster.md

```diff
@@ -355,7 +355,7 @@ Optional:
 
 Optional:
 
-- `replication_factor` (Number) The number of replicas created by RonDB for high availability. Defaults to `2`.
+- `replication_factor` (Number) The number of replicas created by RonDB for high availability. Defaults to `1`.
```

Comment:
Is it still `for high availability` if we have it also in non highly available setups?

Reply:
I updated the description